uxarray/grid.py

```diff
@@ -141,11 +152,14 @@ def __init_grid_var_attrs__(self) -> None:
                 if value in self.ds.dims:
                     setattr(self, key, len(self.ds[value]))
 
-    def __from_vert__(self):
-        """Create a grid with one face with vertices specified by the given
-        argument.
+    def __from_vert__(self, dataset):
+        """Create a grid with multiple faces with vertices specified by the
+        given argument. Called by :func:`__init__`.
 
-        Called by :func:`__init__`.
+        Parameters
+        ----------
+        dataset : ndarray, list, tuple, required
+            Input xarray.Dataset or vertex coordinates that form one face.
```

Comment:
I don't think it can ever be an `xarray.Dataset` because of the following control flow in the `Grid`

```Python
if isinstance(dataset, (list, tuple, np.ndarray)):

Reply:
fixed